packages/thirdweb/src/utils/any-evm/zksync/isZkSyncChain.ts

```diff
@@ -6,19 +6,21 @@
     return false;
   }
 
-  const stack = await getChainStack(chain.id).catch(() => {
-    // fall back to checking against these zksync chain-ids
-    if (
-      chain.id === 324 ||
-      chain.id === 300 ||
-      chain.id === 302 ||
-      chain.id === 11124 ||
-      chain.id === 282 || // cronos zkevm testnet
-      chain.id === 388 // cronos zkevm mainnet
-    ) {
-      return "zksync-stack";
-    }
+  // check known zksync chain-ids first
+  if (
+    chain.id === 324 ||
+    chain.id === 300 ||
+    chain.id === 302 ||
+    chain.id === 11124 ||
+    chain.id === 282 || // cronos zkevm testnet
+    chain.id === 388 // cronos zkevm mainnet
+  ) {
+    return true;
+  }
 
+  // fallback to checking the stack on rpc
+  const stack = await getChainStack(chain.id).catch(() => {
+    // dont throw if network check fails
     return "";
   });
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +21 to +27]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->feels weird, why not simply try catch and if it fails you return `false`?




```suggestion
  // fallback to checking the stack on rpc
  try {
    const stack = await getChainStack(chain.id);
    return stack === "zksync-stack";
  } catch {
    // If the network check fails, assume it's not a ZkSync chain
    return false;
  }
```
<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4883?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->